database/src/main/resources/db/migration/V8_0__add_tx_request_view.sql

```diff
@@ -0,0 +1,89 @@
+UPDATE coin_mint SET status = 'QUEUED' WHERE status = 'INSERTED';
+UPDATE coin_mint SET status = 'PENDING' WHERE status LIKE 'PENDING%';
+UPDATE coin_burn SET status = 'QUEUED' WHERE status = 'INSERTED';
+UPDATE coin_burn SET status = 'PENDING' WHERE status LIKE 'PENDING%';
+UPDATE coin_redemption SET status = 'QUEUED' WHERE status = 'INSERTED';
+UPDATE coin_redemption SET status = 'PENDING' WHERE status LIKE 'PENDING%';
+UPDATE address_registration SET status = 'QUEUED' WHERE status = 'INSERTED';
+UPDATE address_registration SET status = 'PENDING' WHERE status LIKE 'PENDING%';
+UPDATE address_dereg SET status = 'QUEUED' WHERE status = 'INSERTED';
+UPDATE address_dereg SET status = 'PENDING' WHERE status LIKE 'PENDING%';
+
+ALTER TABLE tx_request ADD tx_hash TEXT;
+ALTER TABLE tx_request ADD timeout_height BIGINT;
+ALTER TABLE address_registration ADD timeout_height BIGINT;
+ALTER TABLE address_registration ADD updated TIMESTAMPTZ NOT NULL DEFAULT (NOW() AT TIME ZONE 'utc');
+ALTER TABLE address_dereg ADD timeout_height BIGINT;
+ALTER TABLE address_dereg ADD updated TIMESTAMPTZ NOT NULL DEFAULT (NOW() AT TIME ZONE 'utc');
+CREATE VIEW tx_request_view AS
+SELECT DISTINCT
+    coin_mint.uuid,
+    'MINT' AS type,
+    tx_status.tx_hash,
+    coin_mint.status,
+    timeout_height,
+    coin_mint.created,
+    coin_mint.updated
+FROM coin_mint
+LEFT JOIN tx_status ON coin_mint.uuid = tx_status.tx_request_uuid AND tx_status.status = 'COMPLETE'
```

Comment:
probably obvious later but why is this filtered by status = 'COMPLETE' for mint/burn/redemption?